src/daemon/ca/manager.rs

```diff
@@ -1149,6 +1101,112 @@ impl CaManager {
 /// # Publishing
 ///
 impl CaManager {
+    /// Synchronize all CAs with their repositories. Meant to be called by the background
```

Comment:
How do these changes relate to removal of support for 'embedded'?

Reply:
Moved here now that `CaPublisher` could be removed.